packages/front-end/components/Admin/EditOrganization.tsx

```diff
@@ -52,6 +58,59 @@ const EditOrganization: FC<{
       cta={"Update"}
       close={close}
       inline={!close}
+      secondaryCTA={
+        disablable ? (
+          <div className="flex-grow-1">
+            {currentOrg.disabled ? (
+              <button
+                className="btn btn-info"
+                onClick={(e) => {
+                  e.preventDefault();
+                  apiCall<{
```

Comment:
The onClick should be `async` and you should `await` this call to complete, otherwise when you call `onEdit` below it may return organizations endpoint before the enabling happens.  It did this fairly consistently for me.